api/response/schema_query.py

```diff
@@ -159,33 +137,46 @@ def search_response_by_room(
         search_result.append(responses)
         return search_result
 
+    def get_all_reponses(self, info):
+        response = []
+        rooms = RoomModel.query.all()
+        for room in rooms:
+            room_name = room.name
+            room_response = Response.get_query(info).filter_by(room_id=room.id)
+            total_response = room_response.count()
+            all_responses, total_room_resources = Query.get_room_response(
+                self, room_response, room.id)
+            responses = RoomResponse(
+                room_id=room.id,
+                room_name=room_name,
+                total_responses=total_response,
+                total_room_resources=total_room_resources,
+                response=all_responses)
+            response.append(responses)
+        return response
+
     @Auth.user_roles('Admin')
     def resolve_all_room_responses(
-            self, info, page=None, per_page=None, lower_limit=None,
-            upper_limit=None, room=None, **kwargs):
+        self, info, page=None, per_page=None, lower_limit_count=None,
+        upper_limit_count=None, room=None, upper_date_limit=None,
+        lower_date_limit=None, **kwargs
+    ):
         responses = Query.get_all_reponses(self, info)
-        if isinstance(lower_limit, int) and isinstance(upper_limit, int):
-            responses = Query.filter_rooms_by_responses(
-                self, info, upper_limit, lower_limit
+        check_limits(lower_limit_count, upper_limit_count, int)
+        if (isinstance(upper_limit_count, int)
```

Comment:
The same applies here. I do not think we need to do the instance checks manually given the graphene.type() ensures that the data recieved is of type.

Reply:
> The same applies here. I do not think we need to do the instance checks manually given the graphene.type() ensures that the data recieved is of type.

Kindly work on this comment

Reply:
The graphene.type is a sort of frontend check to restrict the data being entered into insomnia.
The isinstance, here, however, runs the code within the if-block depending on if the data is an `int` or `str`.
It therefore is not a validator